RWS-Submission_Guidelines.md

```diff
@@ -1,53 +1,51 @@
 # Related Website Sets Submission Guidelines 
-# Important Notice regarding Set Submissions 
-As Chrome prepares to [ship Related Website Sets](https://groups.google.com/a/chromium.org/g/blink-dev/c/7_6JDIfE1as) to General Availability (targeting a phased roll-out beginning Chrome 113),  we will shift from “testing” to “live” for Related Website Sets submissions. Please note key dates below as they relate to when your submissions will be applied to Stable behavior in Chrome:
-<ul>
-	<li><b>Monday, April 24, 2023</b>: The related_website_sets.JSON file will be cleared.</li>
-<li><b>Tuesday, April 25, 2023</b>: Submissions will be considered “live” submissions (no longer “test” submissions). The related_website_sets.JSON file will begin to be consumed by Chrome to be applied to Chrome 113+ releases.</li>
-</ul>
 
-# Overview
 Related Website Sets ("RWS") provides a framework for developers to declare relationships among sites, to enable limited cross-site cookie access for specific, user-facing purposes. This framework may help user agents, such as the Chrome browser ("Chrome"), to decide when to allow or deny a site access to their cookies when in a third-party context.
 RWS is a [Privacy Sandbox](https://privacysandbox.com/) proposal being incubated in the W3C's [WICG](https://www.w3.org/community/wicg/). For a full overview, consult the [explainer](https://github.com/privacycg/first-party-sets). The Related Website Sets Submission Guidelines ("Guidelines") are put forth by Chrome to define requirements and expectations for sets submitted by developers. Chrome remains committed to pursuing [standardization](https://www.w3.org/standards/) of RWS through engaging with developers, other browser vendors, and other interested parties.
- # Definitions 
-A <b>Related Website Set</b>, or <b>set</b>, is a collection of domains that is subject to the <a href="#set-formation-requirements">formation requirements</a>, has passed the <a href="#set-validation-requirements">validation requirements</a>, and has been successfully submitted to the canonical RWS list. 
 
-A <b>subset</b> is a defined use case within a set. Set members, or domains, will always be part of a subset. 
+## Definitions
+
+A **Related Website Set**, or **set**, is a collection of domains that is subject to the <a href="#set-formation-requirements">formation requirements</a>, has passed the <a href="#set-validation-requirements">validation requirements</a>, and has been successfully submitted to the canonical RWS list. 
```

Comment:
If you feel like making more formatting updates, I think we can use Markdown syntax instead of HTML syntax for the links (here and elsewhere):
```suggestion
A **Related Website Set**, or **set**, is a collection of domains that is subject to the [formation requirements](#set-formation-requirements), has passed the [validation requirements](#set-validation-requirements), and has been successfully submitted to the canonical RWS list. 
```

Reply:
:) I'm on leave, back 19 September, but I'll have a go then (since I've started, I might as well finish!)